packages/design-picker/src/components/index.tsx

```diff
@@ -34,7 +34,10 @@ interface Props {
 const DesignPicker: React.FC< Props > = ( {
 	locale,
 	onSelect,
-	designs = availableDesigns.featured,
+	designs = availableDesigns.featured.filter(
+		// By default, exclude anchorfm-specific designs
+		( design ) => design.features.findIndex( ( f ) => f === 'anchorfm' ) < 0
```

Comment:
Nit: `includes` serves this purpose more accurately.

Reply:
That's what I tried to write in the first place, but it seems like our build configuration doesn't allow for it — I get an error saying

```Property 'includes' does not exist on type '"anchorfm"[]'. Do you need to change your target library? Try changing the `lib` compiler option to 'es2016' or later.```

I also tried to use it on `Array.prototype` and I got the same error

Reply:
We have examples of packages using [ES2017](https://github.com/Automattic/wp-calypso/blob/trunk/packages/composite-checkout/tsconfig.json#L4) or [ESNext](https://github.com/Automattic/wp-calypso/blob/trunk/packages/language-picker/tsconfig.json#L4) as `lib` option.

Reply:
Great to hear! Definitely going to switch to that for next time